### ISSUE TITLE
Eclipse - autcompletion drop down doesn't show up

### ISSUE THREAD

**Issue by annarbt** [nsf/gocode#321]
The autcomplete doesn't show up at all. I get this in the oracle/gocode console
```
...
>> Running: /home/anna/work/bin/oracle -pos=/tmp/_goclipse9033306876870806610/src/describe_temp/describe.go:#8,#8 -format=json describe describe_temp
oracle: /tmp/_goclipse9033306876870806610/src/describe_temp/describe.go:12:1: expected selector or type assertion, found '}'
====================  Starting gocode server:    ====================
   /home/anna/work/bin/gocode -s -sock=tcp
>> Running: /home/anna/work/bin/gocode -sock=tcp set lib-path /home/anna/workspace/helloGo:/home/anna/work
lib-path "/home/anna/workspace/helloGo:/home/anna/work"
>> Running: /home/anna/work/bin/gocode -sock=tcp -f=csv autocomplete /home/anna/workspace/helloGo/src/main/main.go c129
```
GOROOT = /usr/local/go
GOPATH = /home/anna/work
gocode path = /home/anna/work/bin/gocode

**Comment by nsf** [comment]
Sadly I don't use eclipse, so can't help you. Maybe someone else will. Also worth asking eclipse guys. From the log I can only see some oracle error, and nothing suspicious about gocode.

Also since you're running linux (judging by /home paths), I would recommend trying simpler editors to see if something is wrong with gocode or environment setup. For example godit, a very simple editor: https://github.com/nsf/godit. Type `C-x C-a` for gocode autocompletion, works only on .go files if they are on disk (saved). Type `C-x C-c` for exit. If autocompletion works in godit, then the eclipse setup is the problem.

And finally, try autocompletions on simple code:

```go
package main

import "fmt"

func main() {
  fmt.
}
```

**Comment by bruno-medeiros** [comment]
Yes, first of all, where what is the source of main.go, and where was the cursor when you requested code completion?
